### PR TITLE
feat: apply changes for multiple plans

### PR DESCRIPTION
**Pull request by lovrocolic**

### CODE REVIEW

File: lib/models/batch_event.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
+export default class BatchEvent {
+    constructor(attributes) {
+        this.attributes = attributes
+    }
+
+    wrapAttributes = function () {
+        let event_object = {}
+
+        if (this.attributes.transactionId !== undefined && this.attributes.transactionId !== null)
```
Comment (vincent-pochet) [inline lib/models/batch_event.js:9]: It might worth it to add a helper for this as it is repeated on many places

Comment (vincent-pochet) [reply lib/models/batch_event.js:9]: Also, what if we want ti send a null value for the field?

**Review by vincent-pochet — COMMENTED** [review]
Looks good, only few comments :rocket:

File: lib/models/batch_event.js (hunk 1 above)
Comment (rsempe) [reply lib/models/batch_event.js:9]: Good point - Let's plan to extract them into an helper on a separate PR.

Comment (rsempe) [reply lib/models/batch_event.js:9]: `transaction_id` is a mandatory field

**Review by rsempe — APPROVED** [review]

**Review by vincent-pochet — APPROVED** [review]

**Pull request merged**